### PR TITLE
Added York Peppermint Pattie to no-barcode list, closes #6

### LINKED ISSUE #6: Add peppermint patties to no barcode list
The people want their chocolatey mints!

### PR DESCRIPTION
**Pull request by alex-bellon**
Yes the singular is actually spelled like that and not "Patty" :/

### CODE REVIEW

**Comment by djrenren** [comment]
Yo I hate that spelling

File: web-frontend/pos/mode.js [hunk 1]
```diff
@@ -172,6 +172,7 @@ export class NoBarcodePriceCheck extends Session {
     <h1>Small Snacks</h1>
     <section>
       <button data-barcode="697941861007" onclick="window.mode.select_item(event)">Madeline Cookie</button>
+      <button data-barcode="034000066421" onclick="window.mode.select_item(event)">York Peppermine Pattie</button>
```
Comment (djrenren) [inline web-frontend/pos/mode.js]: But I hate `Peppermine` more. (this is a joke, thank you for your contribution,  pls fix)

Comment (alex-bellon) [reply web-frontend/pos/mode.js]: lol oops B) fix incoming

**Comment by djrenren** [comment]
Okay so I pulled it down just to check it all worked and realized I forgot to tell you about a thing. We need to add Peppermint Pattie to the `manual-barcodes.js` database seed so that the button works for local development.

We just need to add a definition for peppermint patties here:
https://github.com/chezbob/chezbob/blob/9774593d272d6f63dc5cb5c3627e35746a184467/server/db/seeds/manual-barcodes.js#L1

Something like: 

```js
  "York Peppermint Pattie": {
    cents: 25,
    barcode: "034000066421",
  },
```

Basically the reason this seed exists is so that all products referred to in the code are guaranteed to exist in the database.
Yeah, it's super dumb that this stuff is all spread out, (see https://github.com/chezbob/chezbob/issues/7)

After you make that change, you should run: `npm run knex -- seed:run --specific manual-barcodes.js` (it may spit out a bunch of sqlite errors but that's fine). 

You'll know it worked if you spin up chezbob and the price check button for it works.
Sorry this is a pain.
